### PR TITLE
Monitor docs

### PR DESCRIPTION
**Pull request by drewbailey**
docs for monitor command and api, add monitor to changelog

### CODE REVIEW

File: website/source/api/agent.html.md [hunk 1]
```diff
@@ -498,3 +498,62 @@ $ curl \
...
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `agent:read` |
+
+
+### Parameters
+
+- `log-level` `(string: "info")` - Specifies a text string containing a log level
+  to filter on, such as `info`.
```
Comment (cgbaker) [inline website/source/api/agent.html.md]: should we delineate the acceptable values?

File: website/source/api/agent.html.md [hunk 2]
```diff
@@ -498,3 +498,62 @@ $ curl \
...
+
+- `json` `(bool: false)` - Specifies if the log format for streamed logs should be JSON.
+
+- `node-id` `(string: "a57b2adb-1a30-2dda-8df0-25abb0881952")` - Specifies a text
+  string containing a node-id to target for streaming.
```
Comment (cgbaker) [inline website/source/api/agent.html.md]: does this need to be the fully-qualified UUID?

File: website/source/docs/commands/monitor.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+The `nomad monitor` command streams log messages for a given agent.
+
+## Usage
+
+```plaintext
+nomad monitor [options]
+```
+
+The nomad monitor command can be used to stream the logs of a running Nomad agent. Monitor will follow logs and exit when interrupted or until the remote agent quits.
```
Comment (cgbaker) [inline website/source/docs/commands/monitor.html.md.erb]: i think the policy is to limit these lines to 80 characters, but i can't actually find the policy...

Comment (drewbailey) [reply website/source/docs/commands/monitor.html.md.erb]: :+1: will update

**Pull request merged**